Microsoft.Toolkit.Uwp.UI.Controls/RadialGauge/RadialGauge.cs

```diff
@@ -47,13 +47,13 @@ public class RadialGauge : Control
         /// Identifies the Maximum dependency property.
```

Comment:
Not sure if this was broken before, but confused:

![image](https://user-images.githubusercontent.com/24302614/39594118-9cdd082a-4ec0-11e8-94a2-344375a5f460.png)

I'd expect it to snap to 100 or 110...?

Reply:
Yup in the current sample app this works, so something broke:

![image](https://user-images.githubusercontent.com/24302614/39594217-de81d864-4ec0-11e8-84ce-9ef62806efa1.png)

Reply:
I set the initial Value to 105, so the actual value needs to snap to the step size.  I had a case where I wasn't seeing it properly round to 110.  But on my other machine with your same code it seems to be working ok, so I'm not sure what was going on.